### ISSUE TITLE
Broken: e.newsletters.cnn.com

### ISSUE THREAD

**Issue by will-stone** [DrKain/tidy-url#22]
**Website you would like added. Include multiple URLs if needed**

- https://e.newsletters.cnn.com

**Bad parameters. Leave blank if you don't know.**

- click

**Additional context**
Hi @DrKain 👋 I had [this issue](https://github.com/will-stone/browserosaurus/issues/555) filed on Browserosaurus, and it looks like this:

```
https://e.newsletters.cnn.com/click?EcGF1bC5oZXR6ZWxAZ21haWwuY29t/CeyJtaWQiOiIxNjU3NzA5MDUzOTU5ODMzOGNiOGU0Y2E2IiwiY3QiOiJjbm4tYWExNmNkODJiMWViOTY3M2M2ZTYwOTNkZTYyOGE0MGEtMSIsInJkIjoiZ21haWwuY29tIn0/HWkhfQ05OX2lfTmV3c19OREJBTjA3MTMyMDIyNjQ4NTExMSxjbjEsaHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g/qP3Y9YlRJY1FNd1lDMW8/stfbfa11358
```

is turned into this:

```
https://e.newsletters.cnn.com/click?EcGF1bC5oZXR6ZWxAZ21haWwuY29t%2FCeyJtaWQiOiIxNjU3NzA5MDUzOTU5ODMzOGNiOGU0Y2E2IiwiY3QiOiJjbm4tYWExNmNkODJiMWViOTY3M2M2ZTYwOTNkZTYyOGE0MGEtMSIsInJkIjoiZ21haWwuY29tIn0%2FHWkhfQ05OX2lfTmV3c19OREJBTjA3MTMyMDIyNjQ4NTExMSxjbjEsaHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g%2FqP3Y9YlRJY1FNd1lDMW8%2Fstfbfa11358=
```

resulting in the website not loading correctly.

**Comment by DrKain** [comment]
Hi Will, thanks for reporting. 

This one was tricky to solve because any manipulation of [URLSearchParams](https://stackoverflow.com/questions/45516070/urlsearchparams-does-not-return-the-same-string-as-found-in-a-urls-parameters) would encode characters like `/`, even though it probably shouldn't. CNN using url-unsafe b64 probably doesn't help.  

For now I can resolve the issue by using `URL.search`, however this is still encoded when a parameter from URLSearchParams is modified. I'll work on a better solution in another update.

**Comment by DrKain** [comment]
Fixed in `1.5.3`

**Comment by will-stone** [comment]
Thanks @DrKain! 🙂 I'll include it in the next update of Browserosaurus.
